packages/conf/syspar/syspar.go

```diff
@@ -163,6 +169,10 @@ func GetNode(wallet int64) *FullNode {
 	return nil
 }
 
+func GetNodes() map[int64]*FullNode {
```

Comment:
i doesn't think its a good idea, first because mutex is private to that package, and so you expose nodes to some 
other package even without sync primitive at all. If you really need get nodes just for reading make a new map, then copy(with mutex) nodes to that map, and then return

Reply:
Implemented the copying of the map and its returning without a pointer